ast/functions_test.go

```diff
@@ -34,7 +34,7 @@ func (ts *testFunctionsSuite) TestAggregateFuncExtractor(c *C) {
 
 	extractor = &AggregateFuncExtractor{}
 	expr = &FuncCallExpr{
-		FnName: model.NewCIStr("DATE_ARITH"),
+		FnName: model.NewCIStr("FAKE_FUNC"),
```

Comment:
By my understanding, the `expr` here is just a fake function expression for the test, no matter what name it has. Please correct me if I'm wrong :)